packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -37,14 +77,38 @@ class Scrollable extends StatefulWidget {
        assert(viewportBuilder != null),
        super (key: key);
 
+  /// The direction in which this widget scrolls.
```

Comment:
It's not clear what this means. The property is used to map from input gestures to scroll offset deltas?